users/config.rst

```diff
@@ -48,22 +50,27 @@ located:
 
 The database is stored either in the same directory as the config (usually the
 default), but may also be located in one of the following directories (Unix-like
-platforms only):
+platforms only) for legacy reasons:
```

Comment:
We should mention the legacy *config* location as well, i.e. `%XDG_CONFIG_HOME/syncthing` and `$HOME/.config/syncthing`, as they are is still tried.